src/main/java/com/hwr/hwrzeiterfassung/controller/OverviewController.java

```diff
@@ -0,0 +1,111 @@
+package com.hwr.hwrzeiterfassung.controller;
+
+import com.hwr.hwrzeiterfassung.database.controller.LoginController;
+import com.hwr.hwrzeiterfassung.database.repositorys.DayRepository;
+import com.hwr.hwrzeiterfassung.database.repositorys.TimeRepository;
+import com.hwr.hwrzeiterfassung.response.models.DayOverviewCompact;
+import com.hwr.hwrzeiterfassung.response.models.DayOverviewFull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Optional;
+
+@Controller
+@RequestMapping(path = "/overview")
+public class OverviewController {
+    @Autowired
+    private LoginController loginController;
+    @Autowired
+    private DayRepository dayRepository;
+    @Autowired
+    private TimeRepository timeRepository;
+
+    @GetMapping(path = "/DayFullOverviewInInterval")
+    public @ResponseBody
+    Iterable<DayOverviewFull> getDayFullOverviewsInInterval(@RequestParam String email, @RequestParam String password, @RequestParam LocalDate start, @RequestParam LocalDate end) {
```

Comment:
Geht das mit LocalDate ohne Konvertieren? Ich bin da gerade ein wenig am verzweifeln, weil ich immer einen Bad Request zurückbekomme, `Parse attempt failed for value [2022-03-01]` (ich gebe das ohne Klammern an)